upload.py

```
@@ -1230,6 +1283,21 @@ async def update_notification(base_dir: str) -> Optional[str]:
+    # editor (or manual file edits between runs) are picked up.  The
+    # module-level ``config`` dict is imported once at startup and would
+    # otherwise remain stale for the lifetime of the process.  Updating
+    # in-place (clear + update) keeps all existing references (Args,
+    # Clients, managers, etc.) pointing at the same dict object.
+    try:
+        import importlib
+        import data.config as _cfg_mod  # may already be cached
+        importlib.reload(_cfg_mod)
+        config.clear()
+        config.update(_cfg_mod.config)
+    except Exception as exc:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)